lib/zold/version_upgrade_manager.rb

```diff
@@ -0,0 +1,48 @@
+require 'semantic'
+
+module Zold
+  # Class to manage data upgrades (when zold itself upgrades).
+  class VersionUpgradeManager
+    # @todo #285:30min Refactor so we don't just assume there's .zoldata
+    #  in the current directory. Possibly something like a config object
+    #  should be introduced.
+    #
+    # @todo #285:30min Communicate the ongoing upgrade and its steps to the user.
+    #  The user should know:
+    #  - If the upgrade is happening.
+    #  - What upgrade scripts are run.
+    #  - On failure there should be a report of what failed.
+    #
+    # @todo #285:30min Write the upgrade manager tests that ensure:
+    #  - Nothing breaks without the version file.
+    #  - The upgrade scripts run when there is a version file and there are pending upgrade scripts.
+    #  - Make sure *only* the correct upgrade scripts run.
+    def self.run
```

Comment:
@botanicus why static methods? Why can't we create an instance of this "manager" and call its methods? See https://www.yegor256.com/2014/05/05/oop-alternative-to-utility-classes.html